doc/CHANGES.md

```diff
@@ -1,3 +1,16 @@
+# develop
+
+## Highlighted Features
+
+* Configure Google Analytics to anonymise their IP address and disable storing
+  browser cookies. (Graeme Porteous)
+
+# Upgrade Notes
+
+* Google Analytics code was extract into `views/application/_ga_code.html.erb`,
```

Comment:
s/extract/extracted/

I think we should rephrase this a little:

```
The Google Analytics code has been extracted into `views/application/_ga_code.html.erb`, and to avoid the GDPR requirement to show cookie opt-in notices, now sets cookie-less tracking. Additionally, we are instructing Google to anonymise visitors IP address using the `anonymizeIp` option. This means that visitors are no longer individually profiled. If you wish to continue individually targeting users, you'll need to override `app/views/application/_ga_code.html.erb` in your theme and set appropriate options.
```

Reply:
That reads much better